test/utils/cryptography/ECDSA.test.js

```diff
@@ -1,104 +1,101 @@
-require('@openzeppelin/test-helpers');
-const { expectRevertCustomError } = require('../../helpers/customError');
-const { toEthSignedMessageHash } = require('../../helpers/sign');
-
+const { ethers } = require('hardhat');
 const { expect } = require('chai');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
 
-const ECDSA = artifacts.require('$ECDSA');
-
-const TEST_MESSAGE = web3.utils.sha3('OpenZeppelin');
-const WRONG_MESSAGE = web3.utils.sha3('Nope');
-const NON_HASH_MESSAGE = '0x' + Buffer.from('abcd').toString('hex');
+const TEST_MESSAGE = ethers.id('OpenZeppelin');
+const WRONG_MESSAGE = ethers.id('Nope');
+const NON_HASH_MESSAGE = '0xabcd';
 
-function to2098Format(signature) {
-  const long = web3.utils.hexToBytes(signature);
-  if (long.length !== 65) {
-    throw new Error('invalid signature length (expected long format)');
-  }
-  if (long[32] >> 7 === 1) {
-    throw new Error("invalid signature 's' value");
-  }
-  const short = long.slice(0, 64);
-  short[32] |= long[64] % 27 << 7; // set the first bit of the 32nd byte to the v parity bit
-  return web3.utils.bytesToHex(short);
+function toERC2098Format(signature) {
+  return ethers.Signature.from(signature).compactSerialized;
 }
```

Comment:
I think this abstraction is no longer needed and perhaps we can replace with a simple:

```js
function toSignature(signature) {
  return ethers.Signature.from(signature);
}
```

This way we can access `r, s, v` more easily without the `split` function.